<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="zh">
<head>
<!-- Generated by javadoc (1.8.0_162) on Fri Aug 17 17:56:32 CST 2018 -->
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>org.ffpy.validator.exception (Validator)</title>
<meta name="date" content="2018-08-17">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
<script type="text/javascript" src="../../../../script.js"></script>
</head>
<body>
<h1 class="bar"><a href="../../../../org/ffpy/validator/exception/package-summary.html" target="classFrame">org.ffpy.validator.exception</a></h1>
<div class="indexContainer">
<h2 title="异常错误">异常错误</h2>
<ul title="异常错误">
<li><a href="ValidateFailException.html" title="org.ffpy.validator.exception中的类" target="classFrame">ValidateFailException</a></li>
<li><a href="ValidatorException.html" title="org.ffpy.validator.exception中的类" target="classFrame">ValidatorException</a></li>
</ul>
</div>
</body>
</html>
